### ISSUE TITLE
selenium-standalone 3.8.0 from googleapis appears to be corrupted

### ISSUE THREAD

**Issue by krotscheck** [angular/webdriver-manager#289]
I recently started getting odd errors while running protractor. 
```
[15:18:48] E/launcher - Error: Error: Server terminated early with status 1
    at earlyTermination.catch.e (/node_modules/selenium-webdriver/remote/index.js:252:52)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
[15:18:48] E/launcher - Process exited with error code 100
error Command failed with exit code 100.
```

A bit of digging showed that the output to `java -jar selenium-server-standalone-3.8.0.jar` resulted in `Error: Invalid or corrupt jarfile`. Downloading a fresh version from https://selenium-release.storage.googleapis.com/ resulted in the same.

The problem may be worked around using `webdriver-manager update --versions.standalone 3.7.0`

**Comment by nanichandu84** [comment]
I am facing the same issue. Any help would be appreciated.

**Comment by joshzamor** [comment]
Same, though @krotscheck you're workaround didn't seem to work for me.  It still seems to be looking for 3.8.0.

**Comment by nanichandu84** [comment]
@krotscheck even if we specify the version 3.7.0, when running webdriver-manager start, it is still looking for 3.8.0 jar.
Any work around for start to pick up 3.7.0.jar?
Any help would be great.

**Comment by krotscheck** [comment]
Well, all I can offer is the scripts block from our `package.json`:
```
  "scripts": {
    "pree2e": "webdriver-manager update --versions.standalone 3.7.0",
    "e2e": "protractor ./protractor.conf.js"
  },
```
This works for us, YMMV.

**Comment by krotscheck** [comment]
This might also help: https://github.com/angular/webdriver-manager/blob/master/docs/versions.md

**Comment by rongshen** [comment]
Same issue for us.
Our workaround:
webdriver-manager update --versions.standalone=3.7.1
webdriver-manager start --versions.standalone=3.7.1

**Comment by keithrz** [comment]
[chrome #01-47] SessionNotCreatedError: session not created exception: Chrome version must be >= 60.0.3112.0
[chrome #01-47]   (Driver info: chromedriver=2.33.506092 
[chrome #01-47] Command duration or timeout: 2.47 seconds
[chrome #01-47] Build info: version: '3.7.0', revision: '2321c73'

I tried both selenium versions 3.7.0 and 3.7.1.  I probably need to specify a versions.chrome as well...

**Comment by rongshen** [comment]
Ah right, our chrome version is 62

**Comment by keithrz** [comment]
Right, versions.chrome just specifies the chromedriver:
- already 2.33 by default
- supports 60-62

My issue is with my own Docker setup.  chromium-browser is not updating (yay)

**Comment by nanichandu84** [comment]
Thanks @rongshen, that worked like a charm.

Any idea when we can expect resolution for original 3.8.0 jar problem.

**Comment by cnishina** [comment]
Closing this issue. This appears to have been resolved via comments to pin 3.8.0. Thanks everyone.
